### DISCUSSION TITLE
Is it possible to get raw search result count (just a number without actual content) instead of TopK count

### DISCUSSION THREAD

**Discussion by luliloveshe** [milvus-io/milvus#26800] [Q&A and General discussion]
Is it possible to get raw search result count (just a number without actual content) instead of TopK count

**Comment by xiaofan-luan** [comment]
what about doing range search  with iterator

**Reply by luliloveshe** [reply]
I do not quite understand. Can you provide sample code? Thanks

**Reply by xiaofan-luan** [reply]
please see https://milvus.io/docs/within_range.md#Conduct-a-Range-Search 
so far you need to iterate and get all ids and count yourself.
But this is a actually a good suggestion to support count after range search

**Comment by luliloveshe** [comment]
Thank you so much
